documentation/options.md

```diff
@@ -0,0 +1,42 @@
+Plugin options
+===============
+
+You can specify a few options to the plugin:
+
+```JavaScript
+new RailsManifestPlugin({
+  fileName: 'manifest.json',
+  writeToFileEmit: false,
+  extraneous: null,
+  mapAssetPath: (requirePath) => requirePath
+});
+```
+
+### fileName
+
+This is the output file that will be created after each change. The default is `manifest.json`.
+
+### writeToFileEmit
+
+When using this plugin with [webpack-dev-plugin](#) manifest will not be written out to the filesystem, but to in-memory filesystem of the webpack-dev-plugin.
+
+### extraneous
+
+If for any reason you wish to supply the files that will not go thru the Webpack comilation you can do so here. It's just important to note that they **will not** be copied over to the output folder.
```

Comment:
thru -> through
comilation -> compilation
Remove the **just** from just important